docs/modules/publisher-migrationpath.md

```diff
@@ -1,9 +1,9 @@
 [Home](../../readme.md)
 
-# Migration path for publisher version 2.5.x to 2.8.2
+# Migration path for OPC Publisher v2.5.x to v2.8.2
```

Comment:
Let's not use `v` in front of OPC Publisher version numbers. Our tags are without any prefixes.

Reply:
I was using for version short form only and not for tag names. Let me rephrase it.